QRCoder/PayloadGenerator.cs

```diff
@@ -1877,6 +1877,20 @@ public class CalendarEvent : Payload
             private readonly string subject, description, location, start, end;
             private readonly EventEncoding encoding;
 
+            /// <summary>
+            /// Generates a calender entry/event payload.
+            /// </summary>
+            /// <param name="subject">Subject/title of the calender event</param>
+            /// <param name="description">Description of the event</param>
+            /// <param name="location">Location (lat:long or address) of the event</param>
+            /// <param name="start">Start time (incl. UTC offset) of the event</param>
+            /// <param name="end">End time (incl. UTC offset) of the event</param>
+            /// <param name="allDayEvent">Is it a full day event?</param>
+            /// <param name="encoding">Type of encoding (universal or iCal)</param>
+            public CalendarEvent(string subject, string description, string location, DateTimeOffset start, DateTimeOffset end, bool allDayEvent, EventEncoding encoding = EventEncoding.Universal) : this(subject, description, location, DateTime.SpecifyKind(start.ToUniversalTime().DateTime, DateTimeKind.Utc), DateTime.SpecifyKind(end.ToUniversalTime().DateTime, DateTimeKind.Utc), allDayEvent, encoding)
```

Comment:
```suggestion
            public CalendarEvent(string subject, string description, string location, DateTimeOffset start, DateTimeOffset end, bool allDayEvent, EventEncoding encoding = EventEncoding.Universal) : this(subject, description, location, start.UtcDateTime, end.UtcDateTime, allDayEvent, encoding)
```

This syntax performs identically and is cleaner.  The date/time value is converted to UTC, and the produced `DateTime` value has a `Kind` property of `Utc`.